internal/tiltfile/version/version.go

```diff
@@ -1,16 +1,24 @@
 package version
 
 import (
+	"fmt"
+
+	"github.com/mcuadros/go-version"
```

Comment:
would it be reasonable to use blang/semver, since we already use it for docker? worried about having multiple competing semver parsers

Reply:
also, https://github.com/mcuadros/go-version says that it's readonly/archived, which does not give me a lot of confidence that it's well-supported

Reply:
> would it be reasonable to use blang/semver, since we already use it for docker?

yep, I didn't spot that we had that when I was looking for existing version deps